domainlab/mk_exp.py

```diff
@@ -12,8 +12,13 @@
 
 def mk_exp(task, model, trainer, test_domain, batchsize):
     """
```

Comment:
in the current version of our code, argument [trainer] is just a string, but model is a domainlab object. We should put the type of those arguments also in the documentation, either in the form of string, or use

mk_exp(task, model, trainer: str, test_domain:str, batchsize:int)